### ISSUE TITLE
numbers row

### ISSUE THREAD

**Issue by runtav-gyz** [SebastianLuebke/improved-osk-gnome-ext#4]
Hello, this is not really an issue. More like a feature request...

Would it be possible to have the numbers row always visible, just like on a normal keyboard?
Typing passwords with numbers is a constant switching between layouts...

Thank you.

**Comment by luebke-dev** [comment]
yeah, that will be added for sure. im currently trying to make the keyboard more configurable. i will push it at the end of next week.

**Comment by luebke-dev** [comment]
Seems like that is not possible right now. The keyboard breaks if i override the functions needed to do that :(
